docs/other-topics/other-data-types.mdx

```diff
@@ -27,61 +27,61 @@ A ❌ means the dialect does not support that DataType.
-| [`CHAR`](pathname:///api/v7/interfaces/DataTypes.CharDataTypeConstructor.html)     | [`CHAR(255)`][postgres-char]                                    | [`CHAR(255)`][mariadb-char]        | [`CHAR(255)`][mysql-char]    | [`CHAR(255)`][mssql-char]      | `CHAR(255)`                | `CHAR(255)`                                                                   | `CHAR(255)`                 | `CHAR(255)`     |
-| `CHAR(100)`                                                                        | `CHAR(100)`                                                     | `CHAR(100)`                        | `CHAR(100)`                  | `CHAR(100)`                    | `CHAR(100)`                | `CHAR(100)`                                                                   | `CHAR(100)`                 | `CHAR(100)`     |
-| `CHAR.BINARY`                                                                      | `BYTEA`                                                         | `CHAR(255) BINARY`                 | `CHAR(255) BINARY`           | `CHAR(255) BINARY`             | `CHAR BINARY(255)`         | `CHAR(255) BINARY`                                                            | `CHAR(255) BINARY`          | `CLOB(255)`     |
-| `CHAR(100).BINARY`                                                                 | ❌                                                               | `CHAR(100) BINARY`                 | `CHAR(100) BINARY`           | `CHAR(100) BINARY`             | `CHAR BINARY(100)`         | `CHAR(100) BINARY`                                                            | `CHAR(100) BINARY`          | `CLOB(100)`     |
-| `CITEXT`                                                                           | [`CITEXT`](https://www.postgresql.org/docs/current/citext.html) | ❌                                  | ❌                            | ❌                              | `TEXT COLLATE NOCASE`      | ❌                                                                             | ❌                           | ❌               |
-| `TSVECTOR`                                                                         | [`TSVECTOR`][postgres-tsvector]                                 | ❌                                  | ❌                            | ❌                              | ❌                          | ❌                                                                             | ❌                           | ❌               |
+| Sequelize DataType                                                                 | PostgreSQL                                                      | MariaDB                            | MySQL                        | MSSQL                          | [SQLite][sqlite-datatypes] | [Snowflake](https://docs.snowflake.com/en/sql-reference/data-types-text.html) | db2                         | ibmi            | Oracle           |

```

Comment:
Can you add links to the Oracle DataTypes documentation like for the other dialects?